### DISCUSSION TITLE
How to know that i been synced?

### DISCUSSION THREAD

**Discussion by ariz66** [erigontech/erigon#5521] [Q&A]
How to know that i've been synced? this is my log (journalctl)

-- Logs begin at Thu 2022-09-22 22:09:46 CEST. --
Sep 26 06:43:49 srv21540 erigon[5382]: [INFO] [09-26|06:43:49.507] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=3.9GB sys=10.4GB
Sep 26 06:44:49 srv21540 erigon[5382]: [INFO] [09-26|06:44:49.478] [p2p] GoodPeers                          eth66=102
Sep 26 06:44:49 srv21540 erigon[5382]: [INFO] [09-26|06:44:49.488] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=2.9GB sys=10.4GB
Sep 26 06:45:49 srv21540 erigon[5382]: [INFO] [09-26|06:45:49.821] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=3.4GB sys=10.4GB
Sep 26 06:46:49 srv21540 erigon[5382]: [INFO] [09-26|06:46:49.478] [p2p] GoodPeers                          eth66=102
Sep 26 06:46:49 srv21540 erigon[5382]: [INFO] [09-26|06:46:49.493] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=3.1GB sys=10.4GB
Sep 26 06:47:49 srv21540 erigon[5382]: [INFO] [09-26|06:47:49.490] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=3.9GB sys=10.4GB
Sep 26 06:48:49 srv21540 erigon[5382]: [INFO] [09-26|06:48:49.478] [p2p] GoodPeers                          eth66=102
Sep 26 06:48:49 srv21540 erigon[5382]: [INFO] [09-26|06:48:49.488] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=2.9GB sys=10.4GB
Sep 26 06:49:49 srv21540 erigon[5382]: [INFO] [09-26|06:49:49.504] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=3.4GB sys=10.4GB
Sep 26 06:50:49 srv21540 erigon[5382]: [INFO] [09-26|06:50:49.478] [p2p] GoodPeers                          eth66=102
Sep 26 06:50:49 srv21540 erigon[5382]: [INFO] [09-26|06:50:49.857] [txpool] stat                            block=14999999 pending=9930 baseFee=0 queued=30000 alloc=2.9GB sys=10.4GB

**Comment by AskAlexSharov** [comment]
1. can call eth_syncing
2. i see in logs that you seems do not progressing - likely you don't have installed consensus layer.

**Reply by ariz66** [reply]
how to use it on linux terminal? i'm a newbie in this

**Reply by AskAlexSharov** [reply]
1. Let me google it for you https://duckduckgo.com/?q=eth_syncing+example&t=iphone&ia=web
2. If you have no Consensus Layer - search “prysm eth project” or analogs

**Reply by ariz66** [reply]
1. Let me google it for you https://duckduckgo.com/?q=eth_syncing+example&t=iphone&ia=web
Hi, i cannot find it, how to implement on linux? i still don't understand how to use eth_syncing on erigon
i find on geth but not on erigon

2. If you have no Consensus Layer - search “prysm eth project” or analogs
I've installed prysm, and this is my latest log

-- Logs begin at Thu 2022-09-22 22:09:46 CEST. --
Sep 27 03:50:08 srv21540 erigon[5382]: [INFO] [09-27|03:50:08.068] [7/16 Execution] Executed blocks         number=15029839 blk/s=4.5 tx/s=795.9 Mgas/s=68.4 gasState=0.84 estimated duration=12h18m16.784s batch=913.9MB alloc=19.0GB sys=26.2GB
Sep 27 03:50:28 srv21540 erigon[5382]: [INFO] [09-27|03:50:28.073] [7/16 Execution] Executed blocks         number=15030120 blk/s=14.0 tx/s=2593.2 Mgas/s=219.4 gasState=0.85 estimated duration=12h17m57.485s batch=922.2MB alloc=20.9GB sys=26.2GB
Sep 27 03:50:48 srv21540 erigon[5382]: [INFO] [09-27|03:50:48.132] [7/16 Execution] Executed blocks         number=15030327 blk/s=10.3 tx/s=1881.3 Mgas/s=156.8 gasState=0.85 estimated duration=12h19m37.130s batch=929.0MB alloc=22.3GB sys=26.2GB
Sep 27 03:50:49 srv21540 erigon[5382]: [INFO] [09-27|03:50:49.478] [p2p] GoodPeers                          eth66=102
Sep 27 03:50:52 srv21540 erigon[5382]: [INFO] [09-27|03:50:52.869] [txpool] stat                            block=14999999 pending=9922 baseFee=0 queued=30000 alloc=20.4GB sys=26.2GB
Sep 27 03:51:08 srv21540 erigon[5382]: [INFO] [09-27|03:51:08.380] [7/16 Execution] Executed blocks         number=15030441 blk/s=5.6 tx/s=1134.3 Mgas/s=89.9 gasState=0.85 estimated duration=12h23m24.740s batch=932.2MB alloc=15.8GB sys=26.2GB
Sep 27 03:51:28 srv21540 erigon[5382]: [INFO] [09-27|03:51:28.369] [7/16 Execution] Executed blocks         number=15030536 blk/s=4.8 tx/s=834.5 Mgas/s=71.5 gasState=0.86 estimated duration=12h27m42.614s batch=935.1MB alloc=16.5GB sys=26.2GB
Sep 27 03:51:48 srv21540 erigon[5382]: [INFO] [09-27|03:51:48.045] [7/16 Execution] Executed blocks         number=15030638 blk/s=5.2 tx/s=910.0 Mgas/s=80.1 gasState=0.86 estimated duration=12h31m38.775s batch=938.1MB alloc=17.3GB sys=26.2GB
Sep 27 03:51:49 srv21540 erigon[5382]: [INFO] [09-27|03:51:49.494] [txpool] stat                            block=14999999 pending=9922 baseFee=0 queued=30000 alloc=17.4GB sys=26.2GB
Sep 27 03:52:08 srv21540 erigon[5382]: [INFO] [09-27|03:52:08.253] [7/16 Execution] Executed blocks         number=15030745 blk/s=5.3 tx/s=867.3 Mgas/s=77.4 gasState=0.86 estimated duration=12h35m44.917s batch=941.3MB alloc=18.1GB sys=26.2GB
Sep 27 03:52:28 srv21540 erigon[5382]: [INFO] [09-27|03:52:28.332] [7/16 Execution] Executed blocks         number=15030891 blk/s=7.3 tx/s=1298.3 Mgas/s=115.9 gasState=0.87 estimated duration=12h38m34.093s batch=946.1MB alloc=19.2GB sys=26.2GB
Sep 27 03:52:48 srv21540 erigon[5382]: [INFO] [09-27|03:52:48.105] [7/16 Execution] Executed blocks         number=15030981 blk/s=4.6 tx/s=791.0 Mgas/s=67.4 gasState=0.87 estimated duration=12h42m50.629s batch=948.8MB alloc=20.0GB sys=26.2GB
Sep 27 03:52:49 srv21540 erigon[5382]: [INFO] [09-27|03:52:49.479] [p2p] GoodPeers                          eth66=102
Sep 27 03:52:49 srv21540 erigon[5382]: [INFO] [09-27|03:52:49.488] [txpool] stat                            block=14999999 pending=9922 baseFee=0 queued=30000 alloc=20.1GB sys=26.2GB
Sep 27 03:53:08 srv21540 erigon[5382]: [INFO] [09-27|03:53:08.210] [7/16 Execution] Executed blocks         number=15031101 blk/s=6.0 tx/s=1131.8 Mgas/s=95.8 gasState=0.87 estimated duration=12h46m15.254s batch=952.1MB alloc=15.2GB sys=26.2GB
Sep 27 03:53:28 srv21540 erigon[5382]: [INFO] [09-27|03:53:28.214] [7/16 Execution] Executed blocks         number=15031215 blk/s=5.7 tx/s=980.0 Mgas/s=86.3 gasState=0.88 estimated duration=12h49m55.376s batch=955.5MB alloc=16.1GB sys=26.2GB
Sep 27 03:53:48 srv21540 erigon[5382]: [INFO] [09-27|03:53:48.153] [7/16 Execution] Executed blocks         number=15031325 blk/s=5.5 tx/s=971.9 Mgas/s=84.4 gasState=0.88 estimated duration=12h53m36.844s batch=959.1MB alloc=17.0GB sys=26.2GB
Sep 27 03:53:49 srv21540 erigon[5382]: [INFO] [09-27|03:53:49.494] [txpool] stat                            block=14999999 pending=9922 baseFee=0 queued=30000 alloc=17.1GB sys=26.2GB
Sep 27 03:54:08 srv21540 erigon[5382]: [INFO] [09-27|03:54:08.129] [7/16 Execution] Executed blocks         number=15031442 blk/s=5.9 tx/s=1061.0 Mgas/s=89.6 gasState=0.88 estimated duration=12h57m7.307s batch=962.8MB alloc=17.9GB sys=26.2GB
Sep 27 03:54:28 srv21540 erigon[5382]: [INFO] [09-27|03:54:28.390] [7/16 Execution] Executed blocks         number=15031556 blk/s=5.6 tx/s=860.2 Mgas/s=84.1 gasState=0.89 estimated duration=13h0m49.924s batch=966.6MB alloc=18.8GB sys=26.2GB
Sep 27 03:54:48 srv21540 erigon[5382]: [INFO] [09-27|03:54:48.283] [7/16 Execution] Executed blocks         number=15031678 blk/s=6.1 tx/s=1038.5 Mgas/s=95.1 gasState=0.89 estimated duration=13h4m5.910s batch=970.6MB alloc=19.8GB sys=26.2GB

**Reply by AskAlexSharov** [reply]
you are progressing ` [7/16 Execution] Executed blocks number=15031101` total amount of blocks: https://etherscan.io

**Reply by ariz66** [reply]
Ah okay, thank you

How about eth_syncing, how to check if the progress done and i have been synced?

Thanks

**Reply by antoni7654** [reply]
Since bsc stopped around half a day for security issues, my node is not synced. have checked with eth_syncing through web3.py and says "False". Do I have to do something to get it synced again? Thanks.

**Comment by nickgs** [comment]
Hey ariz66, 

Just to complete this thread. To use eth_syncing you can use the following command (using curl): 

`curl http://localhost:8545 -X POST -H "Content-Type: application/json"  --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":67}'`

If this returns false you are not syncing. If it is syncing you will get a data structure that tells you some additional information regarding the starting, current, and highest block. You will want to replace `http://localhost:8545` with the ETH1 RPC endpoint. 

For some additional information regarding this Quicknode has a reference page below:
https://www.quicknode.com/docs/ethereum/eth_syncing

👍
